.rubocop.yml

```diff
@@ -1,6 +1,12 @@
 inherit_from: .rubocop_todo.yml
 
+Style/StringLiterals:
```

Comment:
@apotonick apparently you need to tell rubocop that you prefer the double_quotes.....

Reply:
We need to have this rules in meta and inherit from it.

Reply:
@seuros :+1: can you do it??

Reply:
Let's talk about this in the `core` channel